test/modules/TestResultCoordinator/report/TwinCountingReport.cs

```diff
@@ -5,20 +5,23 @@ namespace TestResultCoordinator.Report
 
     class TwinCountingReport<T> : TestResultReportBase
     {
-        public TwinCountingReport(string trackingId, string expectedSource, string actualSource, string resultType, ulong totalExpectCount, ulong totalMatchCount, ulong totalPatches, ReadOnlyCollection<string> unmatchedResults)
+        public TwinCountingReport(string trackingId, string expectedSource, string actualSource, string resultType, ulong totalExpectCount, ulong totalMatchCount, ulong totalPatches, ulong totalDuplicates, ReadOnlyCollection<string> unmatchedResults)
             : base(trackingId, expectedSource, actualSource, resultType)
         {
             this.TotalExpectCount = totalExpectCount;
             this.TotalMatchCount = totalMatchCount;
-            this.TotalPatches = totalPatches;
+            this.TotalPatchesCount = totalPatches;
+            this.TotalDuplicateResultCount = totalDuplicates;
             this.UnmatchedResults = unmatchedResults;
         }
 
         public ulong TotalExpectCount { get; }
 
         public ulong TotalMatchCount { get; }
 
-        public ulong TotalPatches { get; }
+        public ulong TotalPatchesCount { get; }
+
+        public ulong TotalDuplicateResultCount { get; }
```

Comment:
why Twin will be duplicated?

Reply:
I had a test run that failed to generate the report because of duplicates. It is not an expected behavior, it needs investigation, but I wasn't able to reproduce it.